### PR TITLE
Random password on no password auth

### LINKED ISSUE #10517: Disabling user/password authentication breaks organization user creation
STR

1.- Create an organization
2.- Enable Google authentication, disable user/password authentication.
3.- Try to create a user. You'll get a validation error about missing needed password.

cc @xavijam @CartoDB/builder-backend

### PR DESCRIPTION
**Pull request by juanignaciosl**
This fixes #10517

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@javitonino CR this, please.

File: app/controllers/admin/organization_users_controller.rb [hunk 1]
```diff
@@ -51,6 +53,14 @@ def create
     # The error is deferred to display values in the form in the error scenario.
     validation_failure = !soft_limits_validation(@user, params[:user], current_user.organization.owner)
 
+    if (!current_user.organization.auth_username_password_enabled &&
+            !params[:user][:password].present? &&
+            !params[:user][:password_confirmation].present?)
+      dummy_password = generate_dummy_password
+      params[:user][:password] = dummy_password
+      params[:user][:password_confirmation] = dummy_password
```
Comment (javitonino) [inline app/controllers/admin/organization_users_controller.rb]: A bit hacky, but :ok_hand:

File: lib/dummy_password_generator.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module DummyPasswordGenerator
+  def generate_dummy_password
+    (0...15).map { ('a'..'z').to_a[rand(26)] }.join
+  end
+end
```
Comment (javitonino) [inline lib/dummy_password_generator.rb]: Nice refactor :sparkles:

**Comment by juanignaciosl** [comment]
@javitonino next CR (GitHub fix), please.

**Pull request merged**